### PR TITLE
Speed optimizations

### PR DESCRIPTION
**Pull request by shargon**
Only concat transactions one time

### CODE REVIEW

**Comment by shargon** [comment]
I dont know why fail the check. but i think that the code are good :(

**Comment by AshRolls** [comment]
@shargon see @canesin PR here which will fix the CI process https://github.com/neo-project/neo/pull/82

File: neo/Core/TransactionType.cs [hunk 1]
```diff
@@ -8,28 +10,36 @@ public enum TransactionType : byte
...
         /// 用于分发资产的特殊交易
         /// </summary>
+        [ReflectionCache(typeof(IssueTransaction))]
         IssueTransaction = 0x01,
+        [ReflectionCache(typeof(ClaimTransaction))]
         ClaimTransaction = 0x02,
         /// <summary>
         /// 用于报名成为记账候选人的特殊交易
         /// </summary>
+        [ReflectionCache(typeof(EnrollmentTransaction))]
```
Comment (erikzhang) [inline neo/Core/TransactionType.cs]: Need to disable warnings by adding this:
```
#pragma warning disable CS0612
```

Comment (erikzhang) [inline neo/Core/TransactionType.cs]: Need to disable warnings by adding this:
```
#pragma warning disable CS0612
```

File: neo/Core/TransactionType.cs [hunk 2]
```diff
@@ -8,28 +10,39 @@ public enum TransactionType : byte
...
         /// <summary>
         /// 用于报名成为记账候选人的特殊交易
         /// </summary>
+        #pragma warning disable CS0612
+        [ReflectionCache(typeof(EnrollmentTransaction))]
         EnrollmentTransaction = 0x20,
         /// <summary>
         /// 用于资产登记的特殊交易
         /// </summary>
+        #pragma warning disable CS0612
```
Comment (erikzhang) [inline neo/Core/TransactionType.cs]:
```
#pragma warning disable CS0612
```
Just need to add this once at the top of the file.

**Pull request merged**